src/Modules/Authontication/Components/ChangePass/ChangePass.tsx

```diff
@@ -1,8 +1,169 @@
-import React from 'react'
+import React, { useState } from "react";
+import { useForm } from "react-hook-form";
+import { PasswordValidation } from "../../../../constans/VALIDATIONS";
+import axios from "axios";
+import { USERS_URLs } from "../../../../constans/END_POINTS";
+import { toast } from "react-toastify";
 
 export default function ChangePass() {
+  const [showOldPassword, setShowOldPassword] = useState(false);
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues,
+  } = useForm({ mode: "onBlur" });
+
+  const toggleVisibility = (setterFunction: any) => {
```

Comment:
We can change this to the following:
```tsx
  const toggleVisibility = (setterFunction: any) => {
   return ()=> setterFunction((prevState: any) => !prevState);
  };
```


and use it like that:
`   onClick={toggleVisibility(setShowPassword)}`